### ISSUE TITLE
Change preferences dialog to use native tabs

### ISSUE THREAD

**Issue by 4sensegaming** [kamalyaser31/simple-player#2]
The general, audio, YouTube and shortcuts controls should be implemented as a tab list (like e.g. in Windows Explorer file properties dialog) if possible. Like this, when changing the tab, NVDA reads "expanded" after the tab name, which may be confusing for some users. At first, I was expecting to have to expand or collapse branches of a treeview based on this readout.

**Comment by 4sensegaming** [comment]
After further testing, I figured out that the behavior actually works as intended, as General and Audio do have subcategories / sub-branches. This issue is thus no longer relevant. Sorry about the confusion.
